opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -419,6 +419,11 @@ export const assignOrganizationToUser = async (context, user, userId, organizati
   if (!isUserHasCapability(user, SETTINGS_SET_ACCESSES) && isUserHasCapability(user, VIRTUAL_ORGANIZATION_ADMIN)) {
     throw ForbiddenAccess();
   }
+  const targetUser = await storeLoadById(context, user, userId, ENTITY_TYPE_USER);
```

Comment:
I have a concern about this assignOrganizationToUser, I wonder if we shouldn't use another method, or add more checks when the user is an organization admin (without SETTINGS_SET_ACCESSES).